lib/app/features/home/home_widgets/live_now_view/live_now_view.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:score_live/app/custom_widgets/live_match_tile.dart';
+import 'package:score_live/app/features/home/home_widgets/live_now_view/cubit/live_now_view_cubit.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+
+class LiveNowView extends StatelessWidget {
+  const LiveNowView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<LiveNowViewCubit, LiveNowViewState>(
+      builder: (context, state) {
+        if (state.isLoading == true) {
+          return const Center(
+            child: CircularProgressIndicator(
+              color: AppColors.mainThemePink,
+            ),
+          );
+        }
+
+        final liveMatches = state.liveMatchModel;
+
+        if (liveMatches!.isEmpty) {
+          return const SizedBox(
```

Comment:
Consider moving to separated file as `empty_list_placeholder.dart`

Reply:
done